README.md

```diff
@@ -89,7 +89,7 @@ By default the API runs on port 5000. Calls to the API require a valid bearer
 token. Tokens to be accepted can be set using the DM_AUTH_TOKENS environment
 variable (a colon-separated list), e.g.:
 
-```export DM_API_AUTH_TOKENS=myToken1:myToken2```
+```export DM_API_AUTH_TOKENS=myToken```
```

Comment:
In production (and other life environments) we generally have three active tokens - one for dev, one for Jenkins and one for the front-end apps.